CHANGELOG.md

```diff
@@ -2,34 +2,93 @@
 
 All notable, unreleased changes to this project will be documented in this file. For the released changes, please visit the [Releases](https://github.com/mirumee/saleor/releases) page.
 
-
 # Unreleased
 
 ### Breaking changes
+
 - PREVIEW_FEATURE: replace error code `NOT_FOUND` with `CHECKOUT_NOT_FOUND` for mutation `OrderCreateFromCheckout` - #9569 by @korycins
-- Convert IDs from DB to Graphql format in all notification payloads (email plugins and webhook.NOTIFY)- #9388 by @L3str4nge
+
+### Other changes
+
+- Fix filtering product attributes by date range - #9543 by @IKarbowiak
+- Fix for raising Permission Denied when anonymous user calls `checkout.customer` field - #9573 by @korycins
+
+# 3.2.0
+
+### Breaking changes
+
+- Convert IDs from DB to GraphQL format in all notification payloads (email plugins and the `NOTIFY` webhook)- #9388 by @L3str4nge
 - Migrate order id from int to UUID - #9324 by @IKarbowiak
   - Changed the order `id` changed from `int` to `UUID`, the old ids still can be used
-  for old orders.
+    for old orders.
   - Deprecated the `order.token` field, the `order.id` should be used instead.
   - Deprecated the `token` field in order payload, the `id` field should be used
-  instead.
+    instead.
 - Enable JWT expiration by default - #9483 by @maarcingebala
 
 ### Other changes
-- Fix failing `checkoutCustomerAttach` mutation - #9401 by @IKarbowiak
-- Add OpenID Connect Plugin - #9406 by @korycins
-- Add new mutation `orderCreateFromCheckout` - #9343 by @korycins
-- Add `language_code` field to webhook payload for `Order`, `Checkout` and `Customer` - #9433 by @rafalp
-- Add handling webhook payload via GraphQL subscriptions (#9394)  @jakubkuc
-- Fix access to own resources by App - #9425 by @korycins
+
+#### Saleor Apps
+
 - Introduce custom prices - #9393 by @IKarbowiak
   - Add `HANDLE_CHECKOUTS` permission (only for apps)
+- Add subscription webhooks (#9394) @jakubkuc
+- Add `language_code` field to webhook payload for `Order`, `Checkout` and `Customer` - #9433 by @rafalp
 - Refactor app tokens - #9438 by @IKarbowiak
   - Store app tokens hashes instead of plain text.
-- Fix filtering product attributes by date range - #9543 by @IKarbowiak
-- Save images to product media from external URLs - #9329 by @krzysztofwolski
-- Fix for raising Permission Denied when anonymous user calls `checkout.customer` field - #9573 by @korycins
+- Add category webhook events - #9490 by @SzymJ
+- Fix access to own resources by App - #9425 by @korycins
+- Add `handle_checkouts` permission - #9402 by @SzymJ
```

Comment:
I think this one was done by @[korycins](https://github.com/korycins)

Reply:
Thanks, fixed.